### PR TITLE
feat: 🎸 add sort icon

### PR DESCRIPTION
**Pull request by hevele-moda**
The icon will be used for the new mobile filters: https://modaoperandi.atlassian.net/browse/DIS-669

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,10 @@
     "husky": "^3.1.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0",
+    "react-use": "^13.10.0",
     "semantic-release": "^15.13.31",
+    "styled-components": "^4.4.1",
```
Comment (robertmihaiman) [inline package.json]: do we use those?

Comment (dzucconi) [reply package.json]: We use them in the docs: https://github.com/ModaOperandi/icons/blob/fa52f5e3d8f331b5692bc182a159d480801efb94/example/package.json#L11-L13

So they shouldn't be in this package.json.

Was this added in an effort to get the build passing?

Comment (hevele-moda) [reply package.json]: yeah it's used for the example page, see `example/src/App.js`

Comment (hevele-moda) [reply package.json]: > We use them in the docs:
> 
> https://github.com/ModaOperandi/icons/blob/fa52f5e3d8f331b5692bc182a159d480801efb94/example/package.json#L11-L13
> 
> So they shouldn't be in this package.json.
> 
> Was this added in an effort to get the build passing?

Oh ok that makes sense.
I couldn't get `yarn local:preview` working locally that's why I added them. I'll remove these.

Comment (dzucconi) [reply package.json]: Ah yeah; my bad; I need to add an install step in there.

**Comment by dzucconi** [comment]
Sorry, just noticing these icons aren't outlined so they aren't coloring correctly. Zoom in on them when hovering to see.

**Comment by hevele-moda** [comment]
> Sorry, just noticing these icons aren't outlined so they aren't coloring correctly. Zoom in on them when hovering to see.

Good catch, I'll take a look on it.

**Comment by hevele-moda** [comment]
@dzucconi does it look better now?

**Comment by dzucconi** [comment]
Yah looks good. Though note that they looked different before: had the filled pointy bits.

**Comment by hevele-moda** [comment]
> Yah looks good. Though note that they looked different before: had the filled pointy bits.
> 
> 

Right, by looking on the designs I think that fill is not required.
image

**Pull request merged**